tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -164,6 +164,16 @@ describe('Acceptance: addon-smoke-test', function() {
       });
   });
 
+  it('ember addon with linting errors', function() {
+    return copyFixtureFiles('addon/with-linting-errors')
+      .then(function() {
+        return ember(['test']);
+      })
+      .then(function(result) {
+        expect(result.exitCode).to.not.eql(0);
```

Comment:
I tried to do this; `results.output` was undefined, and `results.outputStream` didn't have anything in it that could be used to verify which test exactly failed.